docs/cep/stream-worker-tasks/create-api-endpoint.md

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 15
+title: Create an API Endpoint
+---
+
+Any stream worker can be used as an API endpoint.
+
+## Create an API Endpoint
+
+You can create a REST API endpoint from a stream worker.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Compute > Stream Workers**.
+1. Write and save stream worker as described in [Create a Stream Worker](create-stream-worker.md).
+1. Click **API Endpoint**.
+
+Macrometa displays the cURL (Client URL) command that you can use in the terminal or command line to make an API call and run the query.
+
+![API Endpoint](/img/stream-workers/api-endpoint.png)
+
+## Stream Worker Endpoints
+
+Why would a user want to do this? Can they send info to a stream worker? Receive it from? What is the point of this cURL commend?
```

Comment:
@VikasPatel - I wasn't aware for this new button `API Endpoint` till now. This seems to be on place of `Edge workers` but how the user could use it?
![image](https://github.com/Macrometacorp/docs/assets/62138955/ce1247c6-5493-4eb1-8b18-3f264e1a9162)

Reply:
Some additional info, this button works only at valid stream worker's definition and generates a curl statement for publishing.
I really don't see the real usage of it.

Reply:
@rahulprabhune-macrometa, if it is not useful, then how hard would it be to remove this?

cc @romatastic, @durgagokina

Reply:
It's not consistent in my opinion, the user could ask if this is for `publishing` how could I get for `unpublishing` using the same or another button.